website/docs/docs/deploy/job-notifications.md

```diff
@@ -1,33 +1,77 @@
+    <Lightbox src="/img/docs/deploy/example-email-notification-settings-page.png" width="100%" title="Example of the Email notifications page"/>
+
+### Unsubscribe from email notifications
+1. From the gear menu in the upper right corner, choose **Notification settings**.
+1. On the **Email notifications** page, click **Unsubscribe from all email notifications**. 
+
+## Slack notifications
+
+You can receive Slack alerts by setting up the Slack integration first, then configuring the dbt Cloud notification settings.
+
+:::note 
+If there have been changes to the user roles and you need to move ownership, please [contact support](mailto:support@getdbt.com) and provide the necessary information for them to make this change for you.
+:::
+
+### Prerequisites 
+- You must be an administrator of the Slack workspace to configure Slack notifications in dbt Cloud. 
+
+### Set up the Slack integration
+
+1. Go to your [Personal profile](https://cloud.getdbt.com/settings/profile) page and scroll to the **Linked accounts** section.
+1. In the **Linked accounts** section, find the Slack application and click **Link**.
+   <Lightbox src="/img/docs/dbt-cloud/Link-your-Slack-Profile.png" width="75%" title="Link for the Slack app"/>
+1. Allow dbt Labs to access the Slack workspace. If you are a member of multiple workspaces, you can select the appropriate workspace from the dropdown menu in the upper right corner.
+   <Lightbox src="/img/docs/dbt-cloud/Allow-dbt-to-access-slack.png" width="75%" title="Allow dbt access to Slack"/>
+
+### Configure Slack notifications
+
+1. From the gear menu in the upper right corner, choose **Notification settings**. 
+1. Select **Slack notifications** in the left sidebar. 
+1. Select the **Notification channel** (in the dropdown) you want to send the job run notifications to. 
+    <Lightbox src="/img/docs/deploy/example-notification-slack-channels.png" width="75%" title="Example of the Notification channel dropdown"/>
+1. Select the **Environment** (in the dropdown) for the jobs you want to receive notifications about. 
+1. Click **Edit** to configure the Slack notification settings. Choose one or more of the run statuses (**Succeeds**, **Fails**, **Is Canceled**) for each job you want to receive notifications for.
+1. When you're done with the settings, click **Save**.
+    
+    To send alerts to another Slack channel, select another **Notification channel** from the dropdown, **Edit** those job notification settings, and click **Save**.
+
+    <Lightbox src="/img/docs/deploy/example-slack-notification-settings-page.png" width="100%" title="Example of the Slack notifications page"/>
+
+### Disable the Slack integration
+
+1. Go to your [Personal profile](https://cloud.getdbt.com/settings/profile) page and scroll to the **Linked accounts** section.
+1. Find the Slack application in the **Linked accounts** section, click the trash can icon, and click **Unlink**. The Slack notification settings are no longer active.
```

Comment:
So in this scenario, the slack notification will still show up for other admins, but will not work until one of those admins goes in and edits and re-saves the notification. This is because we send slack notifications using the slack token of the user who last updated the notification. I wonder if we can specify this somehow in less technical terms?

Reply:
added this bit. not sure if i understood you 100% tho. please take a look and lemme know!